### PR TITLE
porting changes done in the template repo to this repo

### PR DESCRIPTION
**Pull request by bast**

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -33,25 +40,25 @@ The schedule includes frequent breaks.
 - date (weekday) from - to
 - date (weekday) from - to
 
-**Day 1 (April 13, Tue)**
```
Comment (rkdarst) [inline content/_index.md:36]: Do we want to adjust these dates yet

**Review by rkdarst — APPROVED** [review]
I'm not sure if the bit amount removing the dates is intentional or not, but this looks good to me.

More updates are coming soon.

**Comment by bast** [comment]
The date removal was accident. I shall adjust this PR.

**Comment by bast** [comment]
Conflicts .... Working on it ...

**Pull request merged**